src/lib/client.ts

```diff
@@ -86,8 +85,10 @@ export class Client {
                   // TODO
                 }
               })
-              const addr: string | null = LocalStorage.getItem("rewardAddress")
-              const addr2: string = addr ?? farmerPublicKey
+              const addr_unchecked: string | null = LocalStorage.getItem("rewardAddress")
+              let addr!: string;
+              if (addr_unchecked) addr = addr_unchecked
+              else console.log("Error: Reward Address should not be null!")
```

Comment:
If you have an error, use `console.error`, not `console.log`

Reply:
Also for readability please always use `if {} else {}` even if you have single instruction in it.